nvflare/app_common/executors/client_api_launcher_executor.py

```diff
@@ -103,6 +103,7 @@ def __init__(
         self._params_exchange_format = params_exchange_format
         self._params_transfer_type = params_transfer_type
         self._config_file_name = config_file_name
+        self._heartbeat_timeout = heartbeat_timeout
```

Comment:
self.heartbeat_timeout is in the very base class TaskExchanger, should we reuse that instead?